ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -510,6 +510,27 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "remainingChildIndex",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRUD"
```

Comment:
Do we need delete permissions for this one?

Reply:
Think it can be removed, if it is only a temporary field you don't usually need `D` permissions